src/popup/components/buttons/BtnIcon.vue

```diff
@@ -113,5 +113,22 @@ export default {
       opacity: 0.5;
     }
   }
+
+  &.simple {
+    &:hover {
+      background-color: transparent;
+    }
+  }
+
+  &.simple-danger {
+    &:hover {
+      background-color: transparent;
+
+      .icon {
+        color: variables.$color-danger;
+        opacity: 1;
+      }
+    }
+  }
```

Comment:
This styles in my opinion are obsolete. The BtnIcon is a component that has one role: adds circular background on hover. Removing this background makes no sense as this component starts to behave like a regular `<a>` without particular styling.